src/tools/fuzzing.h

```diff
@@ -1320,12 +1320,24 @@ class TranslateToFuzzReader {
     return first;
   }
 
+  #define GCC_VERSION (__GNUC__ * 10000 \
```

Comment:
Thanks, this looks good. Please just add a comment before this with an explanation and a link to the upstream bug ( https://gcc.gnu.org/bugzilla/show_bug.cgi?id=82800 ).

Also, before we merge, have you joined the [community group](https://www.w3.org/community/webassembly/participants)?

Reply:
I added the comment and signed up for the community group :)